docs/platform/concepts/authentication-tokens.rst

```diff
@@ -0,0 +1,35 @@
+Authentication tokens
+=====================
+
+An authentication token allows a user to programmatically access resources instead of using their username-password. Unlike the username-password combination that has a longer shelf life (in some case, they never expire); you can limit how long an authentication token is valid for or revoke it completely.
```

Comment:
Should we also state that several tokens can be active at the time, so you can create a token for a specific use case and revoke when the project is finished.

Reply:
Great point. I'll be sure to add that line @ftisiot